src/sentry/testutils/cases.py

```diff
@@ -1376,13 +1376,18 @@ def setUp(self):
         self.addCleanup(patcher.stop)
 
     def drop_replays(self):
-        assert requests.post(settings.SENTRY_SNUBA + "/tests/replays/drop").status_code == 200
+        assert (
+            requests.post(settings.SENTRY_SNUBA + "/tests/entities/replays/insert").status_code
+            == 200
+        )
```

Comment:
This is no longer dropping the table, will that work?

Reply:
ah 🤦🏼

Reply:
you shouldn't have to change this actually, the `drop` endpoint is still there in master

Reply:
yup 👍🏼